myokit/formats/sbml/_importer.py

```diff
@@ -371,39 +1103,43 @@ class SBMLError(myokit.ImportError):
     """
 
 
-unit_map = {
-    'dimensionless': myokit.units.dimensionless,
+# SBML base units according to libSBML
+SBML2MyoKitUnitDict = {
     'ampere': myokit.units.A,
-    'farad': myokit.units.F,
-    'katal': myokit.units.kat,
-    'lux': myokit.units.lux,
-    'pascal': myokit.units.Pa,
-    'tesla': myokit.units.T,
+    'avogadro': None,  # TODO: myokit equivalent not yet looked up
     'becquerel': myokit.units.Bq,
-    'gram': myokit.units.g,
-    'kelvin': myokit.units.K,
-    'meter': myokit.units.m,
-    'radian': myokit.units.rad,
-    'volt': myokit.units.V,
     'candela': myokit.units.cd,
-    'gray': myokit.units.Gy,
-    'kilogram': myokit.units.kg,
-    'metre': myokit.units.m,
-    'second': myokit.units.s,
-    'watt': myokit.units.W,
     'celsius': myokit.units.C,
-    'henry': myokit.units.H,
-    'liter': myokit.units.L,
-    'mole': myokit.units.mol,
-    'siemens': myokit.units.S,
-    'weber': myokit.units.Wb,
     'coulomb': myokit.units.C,
+    'dimensionless': myokit.units.dimensionless,
+    'farad': myokit.units.F,
+    'gram': myokit.units.g,
+    'gray': myokit.units.Gy,
+    'henry': myokit.units.H,
     'hertz': myokit.units.Hz,
-    'litre': myokit.units.L,
-    'newton': myokit.units.N,
-    'sievert': myokit.units.Sv,
+    'item': None,  # TODO: myokit equivalent not yet looked up
     'joule': myokit.units.J,
+    'katal': myokit.units.kat,
+    'kelvin': myokit.units.K,
+    'kilogram': myokit.units.kg,
+    'liter': myokit.units.L,
+    'litre': myokit.units.L,
     'lumen': myokit.units.lm,
+    'lux': myokit.units.lux,
+    'meter': myokit.units.m,
+    'metre': myokit.units.m,
+    'mole': myokit.units.mol,
+    'newton': myokit.units.N,
     'ohm': myokit.units.R,
+    'pascal': myokit.units.Pa,
+    'radian': myokit.units.rad,
+    'second': myokit.units.s,
+    'siemens': myokit.units.S,
+    'sievert': myokit.units.Sv,
     'steradian': myokit.units.sr,
+    'tesla': myokit.units.T,
+    'volt': myokit.units.V,
+    'watt': myokit.units.W,
+    'weber': myokit.units.Wb,
+    'invalid': None,
```

Comment:
14 invalids? doesn't look like a unit to me :D

Reply:
hmm I don't remember why I put it there, and it's not listed in the above base unit reference, so It may just be a mistake. I'll remove it.